### ISSUE TITLE
Don't use private when programming

### ISSUE THREAD

**Issue by clankill3r** [ketai/ketai#54]
Please don't use `private` when programming but use `protected` instead.
That way when a programmer that knows what he is doing extends a class he can still access fields.
Now private things have to be retrieved with reflection...

my 2 cents

**Comment by kfrajer** [comment]
I have to agree with this. A relevant post: https://forum.processing.org/two/discussion/26890/ketailist#latest
Making it protected is not enough. Function calling based on specific action will allow to re-use the code. Pseudo-concept illustrated in the provided link. However, let's start by moving away from private fields.
